src/db/migrations/20220808070221_create_patient_table.ts

```diff
@@ -0,0 +1,20 @@
+import { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable("patient", (table) => {
+    table.increments("id");
+    table.string("first_name").notNullable();
+    table.string("last_name").notNullable();
+    table.string("email").notNullable().unique();
+    table.date("date_of_birth").notNullable();
+    table.string("ethnicity").notNullable();
+    table.string("gender").notNullable();
+    table.json("address").notNullable();
```

Comment:
I don't think it is a good way to store data. It can cause difficulty in querying specific attributes. I'm also confused on whether to use it or not.

Reply:
I was also initially planning to implement address and payment in different tables with 1-to-1 relation but went with this approach for now.